lib/players/human.rb

```diff
@@ -1,6 +1,8 @@
+require_relative '../get_input.rb'
 class Player::Human < Player
+  include GetInput
   def move(_game)
     puts 'Enter board position [1-9]:'
-    gets.chomp
+    get_input
```

Comment:
I would make `get_input` a class method and reference it with `GetInput.get_input` rather than using include, which could pollute the methods under this class.